### PR TITLE
Timeseries forecasting debug

### PR DESCRIPTION
**Pull request by yufeiyu**
#### What type of PR is this?
feature

#### What this PR does / why we need it:
Add  http interface for debugging DSP parameters.

#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

**Comment by kitianFresh** [comment]
Please squash all the commits into just one commit

**Comment by zsnmwy** [comment]
Please make sure all checks are passed.

File: pkg/prediction/dsp/prediction_dubug.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package dsp
```
Comment (lbbniu) [inline pkg/prediction/dsp/prediction_dubug.go:1]: prediction_dubug.go rename prediction_debug.go

File: docs/tutorials/using-time-series-prediction.md [hunk 2]
```diff
@@ -96,12 +96,37 @@ There are three types of the metric query:
...
 
 ### Algorithm
-`Algorithm` define the algorithm type and params to do predict for the metric. Now there are two kinds of algorithms:
-
- - `dsp` is an algorithm to forcasting a time series, it is based on FFT(Fast Fourier Transform), it is good at predicting some time series with seasonality and periods.
- - `percentile` is an algorithm to estimate a time series, and find a recommended value to represent the past time series, it is based on exponentially-decaying weights historgram statistics. it is used to estimate a time series, it is not good at to predict a time sequences, although the percentile can output a time series predicted data, but it is all the same value. so if you want to predict a time sequences, dsp is a better choice.
+`Algorithm` defines the algorithm type and the corresponding parameters. Currently, crane provides two built-in prediction algorithms:
+ - `dsp` borrows some techniques from 'digital signal processing', such as FFT(Fast Fourier Transform), for periodicity identification and time series forecasting. It is good at predicting the time series that follow some cyclic patterns.
```
Comment (zsnmwy) [inline docs/tutorials/using-time-series-prediction.md:100]: Please add a new line between the list title and the list item.
If not, the list style not be effective.
image

**Review by zsnmwy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**